pkg/agent/bakerapi_test.go

```diff
@@ -339,10 +366,18 @@ var _ = Describe("AgentBaker API implementation tests", func() {
 
 		It("should return correct value for existing distro when linux node image version override is provided but not for distro", func() {
 			toggles.Maps = map[string]agenttoggles.MapToggle{
-				"linux-node-image-version": func(entity *agenttoggles.Entity) map[string]string {
+				"vhd-types": func(entity *agenttoggles.Entity) map[string]string {
 					return map[string]string{
-						string(datamodel.AKSUbuntu1804): "202402.27.0",
+						"vhd-type": "override",
+					}
+				},
+				"linux-node-image-version": func(entity *agenttoggles.Entity) map[string]string {
+					if entity.Fields["vhd-type"] == "override" {
```

Comment:
probably don't want references to `vhd-types` in tests if we're not gonna end up using it

Reply:
Removed, but then added it back as part of the split of files.